crates/bevy_render/src/lib.rs

```diff
@@ -112,12 +113,12 @@ impl RenderSet {
         let mut schedule = Schedule::new();
 
```

Comment:
Does this schedule have a default base set?

Reply:
No, I went by Alice's comment:

> I don't think the `RenderSchedule` should have one: there's no clear default.

But I can add one if needed :)

Reply:
Are there any errors that arise if you add a system to a schedule that has base sets but doesn't pick one as default?

Reply:
How would I best test that?

Reply:
It should be as simple as trying `render_schedule.add_system(my_system)`. I'm pretty sure that should error with something like "my_system needs to be put into a base set", but worth checking.

Reply:
It might make sense to change this to only change the startup schedule to use base sets, but move changing the render schedule to a different pr. Feels like there's a missin configuration option to require that a base set be set, but not set a default set.

Reply:
Ok, I reverted the `RenderSet` changes and updated the PR description.